core/templates/pages/profile-page/profile-page.component.ts

```diff
@@ -54,7 +54,6 @@ angular.module('oppia').component('profilePage', {
         let fetchProfileData = () =>
           ProfilePageBackendApiService.fetchProfileData();
         fetchProfileData().then(function(data) {
-          LoaderService.hideLoadingScreen();
```

Comment:
Removed this as we are expecting to hide the loading screen after initializing all the variables. Found this is also done at the end ([line 216](https://github.com/oppia/oppia/pull/9897/files#diff-47275b7a8e7e274c1b0fe7b67d863c50L216)) 

I'm expecting this to fix some flakiness in profileFeature suite.